crates/kubelet/src/server.rs

```diff
@@ -49,9 +49,9 @@ pub async fn start_webserver<T: 'static + Provider + Send + Sync>(
                             get_container_logs(
                                 &*provider.lock().await,
                                 &req,
-                                namespace.to_string(),
-                                pod.to_string(),
-                                container.to_string(),
+                                (*namespace).to_string(),
```

Comment:
out of curiosity, what's the reason for this change? How come we have to deref here?

Reply:
This is a clippy suggestion. Basically `namespace` is a `&&` which has a slower implementation of to_string

Reply:
Out of curiosity, does `to_owned` lead to the same clippy warning?